pyaedt/modeler/cad/object3d.py

```diff
@@ -1304,7 +1304,7 @@ def display_wireframe(self, fWireframe):
         self._change_property(vWireframe)
         self._wireframe = fWireframe
 
-    @property
+    @pyaedt_function_handler()
```

Comment:
Why is it not a property anymore?

Reply:
it's too slow due to recursiveness and makes both debugging and run very slow

Reply:
Because it is a time and resource intensive operation. Since it gets called every time the object is inspected it was slow to debug the objects (pycharm sometimes goes in timeout).